src/components/sakVelger/SakVelger.tsx

```diff
@@ -0,0 +1,200 @@
+                setValgtTema
+            })}
+        </>
+    );
+};
+
+interface SakVelgerRadioGroupProps {
+    valgtSakKategori: SakKategori;
+    setSakKategori: (sakKategori: SakKategori) => void;
+}
+
+const SakVelgerRadioGroup: React.FC<SakVelgerRadioGroupProps> = ({ valgtSakKategori, setSakKategori }) => {
+    return (
+        <RadioGroup legend="Saktype" value={valgtSakKategori} onChange={setSakKategori}>
+            <HStack gap="2">
+                {Object.values(SakKategori).map((sakKategori) => (
+                    <Radio value={sakKategori} key={sakKategori}>
+                        {sakKategori}
+                    </Radio>
+                ))}
+            </HStack>
+        </RadioGroup>
+    );
+};
+
+interface SakVelgerTemaTableProps {
+    kategorier: Kategorier;
+    valgtKategori: SakKategori;
+    valgtTema: Tema | undefined;
+    setValgtTema: (tema: Tema) => void;
+}
+
+const SakVelgerTemaTable: React.FC<SakVelgerTemaTableProps> = ({
+    kategorier,
+    valgtKategori,
+    valgtTema,
+    setValgtTema
+}) => {
+    return (
+        <Table zebraStripes className="table-fixed">
+            <Table.Header>
+                <Table.HeaderCell scope="col">Tema</Table.HeaderCell>
+            </Table.Header>
+            <Table.Body>
+                {kategorier[valgtKategori].map((tema) => (
+                    <Table.Row
+                        key={tema.tema}
+                        onClick={() => {
+                            setValgtTema(tema);
+                        }}
+                        selected={tema.tema === valgtTema?.tema}
+                        className="cursor-pointer"
+                    >
+                        <Table.DataCell>{tema.tema}</Table.DataCell>
+                    </Table.Row>
+                ))}
+            </Table.Body>
+        </Table>
+    );
+};
+
+interface SakVelgerSakTableProps {
+    kategorier: Kategorier;
+    valgtKategori: SakKategori;
+    valgtTema: Tema | undefined;
+    setSak: (sak: JournalforingSak) => void;
+}
+
+const SakVelgerSakTable: React.FC<SakVelgerSakTableProps> = ({ kategorier, valgtKategori, valgtTema, setSak }) => {
+    return (
+        <Table zebraStripes className="table-fixed">
+            <Table.Header>
+                <Table.HeaderCell scope="col">SaksId</Table.HeaderCell>
+                <Table.HeaderCell scope="col">Opprettet</Table.HeaderCell>
+            </Table.Header>
+            <Table.Body>
+                {kategorier[valgtKategori]
+                    .filter((it) => it.tema === valgtTema?.tema)
+                    .flatMap((it) =>
+                        it.saker.map((sak) => (
+                            <Table.Row
+                                key={sak.saksId}
+                                onClick={() => {
+                                    setSak(sak);
+                                }}
+                                className="cursor-pointer"
+                            >
```

Comment:
En klikkbar table row er ikke spesielt egnet for skjermlesere. Du kan lage en egen kolonne som inneholder en knapp med en beskrivende tittel som ikke er "synlig" slik at det er enklere for skjermlesere å vite hvor de skal "trykke". Vi hadde et eksempel på dette i innkrevingskrav, så du kan evt se hvordan det ble gjort der :)